### PR TITLE
Delete szActualText when done in consoleargs

### PR DESCRIPTION
**Pull request by omajid**

### CODE REVIEW

**Comment by omajid** [comment]
Anyone from @dotnet/crossgen-contrib willing to review this?

File: src/coreclr/tools/util/consoleargs.cpp [hunk 1]
```diff
@@ -953,6 +953,11 @@ void ConsoleArgs::ProcessResponseArgs()
 #endif
 
         TextToArgs(szActualText, &listCurArg->next);
+
+        delete[] pwzFileBuffer;
+#ifndef TARGET_UNIX
+        delete[] szExpandedBuffer;
```
Comment (mangod9) [inline src/coreclr/tools/util/consoleargs.cpp:959]: does this need a null check before deletion?

**Review by janvorli — APPROVED** [review]
LGTM, thank you!

File: src/coreclr/tools/util/consoleargs.cpp (hunk 1 above)
Comment (janvorli) [reply src/coreclr/tools/util/consoleargs.cpp:959]: passing NULL to delete is valid

Comment (janvorli) [reply src/coreclr/tools/util/consoleargs.cpp:959]: See http://www.cplusplus.com/reference/new/operator%20delete/:
> Parameters
ptr
A pointer to the memory block to be released, type-casted to a void*.
If this is a null-pointer, the function does nothing.

**Review by mangod9 — APPROVED** [review]

**Pull request merged**